status-network-contracts/foundry.lock

```diff
@@ -0,0 +1,26 @@
+{
+  "../go-corset": {
+    "rev": "ee34d9eb60bf369958fcaae6b97a0c53502092fc"
+  },
+  "lib/openzeppelin-contracts": {
+    "branch": {
+      "name": "v4.8.3",
+      "rev": "0a25c1940ca220686588c4af3ec526f725fe2582"
+    }
+  },
+  "../constraints": {
+    "rev": "445209f0ad75d5b29b7e6924508dd2579a42508a"
+  },
+  "../contracts/lib/forge-std": {
+    "rev": "1eea5bae12ae557d589f9f0f0edae2faa47cb262"
+  },
+  "lib/forge-std": {
+    "rev": "035de35f5e366c8d6ed142aec4ccb57fe2dd87d4"
+  },
+  "lib/openzeppelin-contracts-upgradeable": {
+    "branch": {
+      "name": "v4.8.3",
+      "rev": "58fa0f81c4036f1a3b616fdffad2fd27e5d5ce21"
+    }
+  }
+}
```

Comment:
Foundry has never created a foundry.lock file for me.
This is useful, but should probably be in a separate commit.

Your changes in the staking system don't actually introduce any new dependencies.

Also any reason why `go-corset` and `constraints` is in there?

Reply:
It was just autogenerated, removed it